### PR TITLE
Rails 7.1 fix: use SelectRenderer for add_options and include FormOptionsHelper for …

### PR DESCRIPTION
**Pull request by hishammalik**
…currency_options_for_select

### CODE REVIEW

File: lib/to_currency_select_tag.rb [hunk 1]
```diff
@@ -10,6 +11,7 @@ module ToCurrencySelectTag
       def to_currency_select_tag(priority_currencies, options, html_options)
         html_options = html_options.stringify_keys
         add_default_name_and_id(html_options)
+        byebug
```
Comment (oliverklee) [inline lib/to_currency_select_tag.rb:14]: This look like a left-over debug statement. Could you please remove it?

**Review by oliverklee — CHANGES_REQUESTED** [review]
Thanks for your contribution! :heart: I've added my review comments.

File: lib/to_currency_select_tag.rb (hunk 1 above)
Comment (hishammalik) [reply lib/to_currency_select_tag.rb]: oops. Fixed

File: lib/to_currency_select_tag.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+require 'form_options_helper'
```
Comment (oliverklee) [inline lib/to_currency_select_tag.rb:2]: RuboCop wants a blank line between the `#frozen_string …` line and the next line.

**Review by oliverklee — CHANGES_REQUESTED** [review]
Getting close … now let’s make RuboCop happy. :-)

File: lib/to_currency_select_tag.rb (hunk 2 above)
Comment (hishammalik) [reply lib/to_currency_select_tag.rb:3]: strange. My rubocop instance didn't complain. I've added it though and pushed 👍

**Review by oliverklee — APPROVED** [review]
Looks good. Thanks a lot, @hishammalik! :heart:

**Comment by oliverklee** [comment]
Sorry, this got merged to the wrong branch, which I accidentally overwrote. I'm going to manually re-apply your changes in a minute. Sorry for this.

@hishammalik  For the future, I'd like to ask you to only do PRs against a project's main branch (which usually is `main` oder `master`) to make sure your changes don't get lost.

**Comment by oliverklee** [comment]
I've re-applied your changes now in #151.

**Comment by oliverklee** [comment]
Hi @hishammalik, thank you so much again for contributing! :heart: I've just released version 6.0.0 with your changes: https://github.com/braingourmets/currency_select/releases/tag/v6.0.0

**Pull request merged**